build/shared-runtime.version

```diff
@@ -1 +1 @@
-1.2.0-beta-001304-00
\ No newline at end of file
+1.1.1;release/1.1.0,2.0.0-beta-001747-00;master
```

Comment:
The mirroring does try to figure out the shared runtime based on the CLI (https://github.com/aspnet/CoreCLR/blob/dev/tools/MirrorLatestCoreCLR/Program.cs#L97), but we stopped using this because Core-Setup wanted us to not rely on them to get the latest shared runtime. Would be much better to install the shared runtime separately.